gulpfile.js

```diff
@@ -894,6 +895,10 @@ function dist() {
         if (argv.fortesting) {
           return enableLocalTesting(minifiedRuntimeTarget);
         }
+      }).then(() => {
+        return createModuleCompatibleES5Bundle('v0.js');
+      }).then(() => {
+        return createModuleCompatibleES5Bundle('amp4ads-v0.js');
```

Comment:
What about shadow doc and web worker entrypoints?

Reply:
we are just replacing these two files in phase1.
rest remain as in.